pkg/utils/json_utils.go

```
@@ -12,27 +12,51 @@
+		return "", err
+	}
 
 	var prettyJSON bytes.Buffer
 	err = json.Indent(&prettyJSON, []byte(j), "", "  ")
 	if err != nil {
-		return err
+		return "", err
+	}
+	highlighted, err := HighlightCodeWithConfig(atmosConfig, prettyJSON.String())
+	if err != nil {
+		return prettyJSON.String(), nil
```

golangci-lint (1.64.8)

[error] error is not nil (line 35) but it returns nil

(nilerr)